doc/source/user_guide/ug_running.rst

```diff
@@ -229,6 +232,8 @@ change the defaults.  This is done as part of the ``cice.setup`` and the
 modifications are resolved in the **cice.settings** and **ice_in** file placed in 
 the case directory.  If multiple options are chosen that conflict, then the last
 option chosen takes precedent.  Not all options are compatible with each other.
+Settings on the command line will take precedent over serttings defined in 
```

Comment:
I think this should be corrected to
"... take precedence over settings ..."

Reply:
Interesting, I had to look it up.  I fixed 5 errors in the document.  Thanks!